### ISSUE TITLE
Nix install displays https://nixos.org SSL certificate expired message.

### ISSUE THREAD

**Issue by nekoniaow** [NixOS/nix#7810]
**Describe the bug**

When trying to run the basic nix install command `$ sh <(curl -L https://nixos.org/nix/install)` (on MacOS although that's likely irrelevant) I get the following error message:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
HTTPS-proxy has similar options --proxy-cacert and --proxy-insecure.
```

**Steps To Reproduce**

1. Open a shell
2. Type `sh <(curl -L https://nixos.org/nix/install)`
3. Notice the error message about the expired SSL certificate.

**Expected behavior**

The install proceeds normally.

**Comment by nixinator** [comment]
could be related to 
https://letsencrypt.org/docs/dst-root-ca-x3-expiration-september-2021/

and

https://apple.stackexchange.com/questions/428448/app-curl-still-getting-certificate-expired-error-due-to-expired-lets-encrypt

**Comment by SuperSandro2000** [comment]
> on MacOS although that's likely irrelevant

I don't think so. If this was not a temporary problem on our side, then this is definitely related to your machine and the ca certificates it has or maybe also to your almost 6 year old curl which might or might not use that old certificates depending on how it was build.

**Comment by abathur** [comment]
I happen to have an old Air collecting dust (same macOS and curl versions) and confirmed that I both see the same when using `sh <(curl -L https://nixos.org/nix/install)` and that I was able to install using the installer URL directly from releases.

**Comment by nekoniaow** [comment]
> > on MacOS although that's likely irrelevant
> 
> I don't think so. If this was not a temporary problem on our side, then this is definitely related to your machine and the ca certificates it has or maybe also to your almost 6 year old curl which might or might not use that old certificates depending on how it was build.

That makes sense, I will give it another try after I update my OS to a more recent tone.
Thanks.

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
